packages/itwin/grouping-mapping-widget/src/widget/components/Grouping.tsx

```diff
@@ -84,6 +87,7 @@ enum GroupsView {
   MODIFYING = "modifying",
   ADD = "ADD",
   PROPERTIES = "properties",
+  CUSTOM = "custom",
```

Comment:
This enum is inconsistent. String values should be formatted similarly (`ADD` is all uppercase while everything else is lowercase). Also, Change enum value names to PascalCasing.